README.md

```diff
@@ -4,7 +4,7 @@
 A plugin for SonarQube to allow branch analysis in the Community version.
 
 # Installation
-Copy the plugin JAR file to the `extensions/plugins/`directory of your SonarQube instance and restart SonarQube.
+Either build the project or [download the RELEASE version of the plugin JAR](https://github.com/mc1arke/sonarqube-community-branch-plugin/releases). Copy the plugin JAR file to the `extensions/plugins/`directory of your SonarQube instance and restart SonarQube.
```

Comment:
Could you change `RELEASE` to `release` please?

Reply:
yep!